src/main/kotlin/org/rust/lang/core/grammar/rust.bnf

```diff
@@ -269,7 +269,6 @@
     extends     ("(if|while)_let_expr") = scoped_let_expr
     extends     (".*_expr") = restricted_expr
     extends     ("(?!ret_type).*_type") = type
-    extends     (".*_stmt") = stmt
```

Comment:
Maybe I'm a bit picky, but this rule seemed fine. Why not just add `|let_decl` to it (`extends     (".*_stmt|let_decl") = stmt`)?